app/helpers/dasherize.js

```diff
@@ -0,0 +1,8 @@
+import { dasherize } from '@ember/string';
+import { helper } from '@ember/component/helper';
+
+export function dasherizeString([str]) {
+  return dasherize(str).replace(/\//g, '::');
+}
+
+export default helper(dasherizeString);
```

Comment:
This helper is unused, do you intend to keep it around?

Reply:
(if so, the `.replace(/\//g, '::')` is incorrect since in curly invocation it _should_ be slashes)

Reply:
Deleted it